hammr/commands/image/image.py

```diff
@@ -135,14 +136,12 @@ def do_publish(self, args):
 
             try:
                 if doArgs.id:
-                    images = self.api.Users(self.login).Images.Getall()
-                    images = images.images.image
-                    if images is None or len(images) == 0:
-                        printer.out("No images available")
-                    else:
-                        for iimage in images:
-                            if str(iimage.dbId) == str(doArgs.id):
-                                image = iimage
+                    images = self.get_all_images()
```

Comment:
Maybe this bloc of code can be isolated in some "get_image" method.
I wonder why we can't call directly a "get" webservices as it seems we have the image id, but It would be more than a refactoring to change it (and maybe this webservices doesn't exist).

Reply:
I have made investigations, and unfortunately it seems that there is no direct "get" webservice for an image.